common/djangoapps/common_initialization/apps.py

```diff
@@ -0,0 +1,66 @@
+"""
+Common initialization app for the LMS and CMS
+"""
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import Error, Tags, register
+
+
+class CommonInitializationConfig(AppConfig):
+    name = 'common_initialization'
+    verbose_name = 'Common Initialization'
+
+
+@register(Tags.compatibility)
+def validate_settings(app_configs, **kwargs):
+    """
+    Common settings validations for the LMS and CMS.
+
+    Only populate this method with general settings validations which do not
+    fit in the validations in other, more specific djangoapps.  Usually,
+    settings which are widely used across the entire LMS or CMS can be
+    validated here.
+    """
+    errors = []
+    errors.extend(_validate_lms_root_url_setting())
+    errors.extend(_validate_marketing_site_setting())
+    return errors
+
+
+def _validate_lms_root_url_setting():
+    """
+    Validates the LMS_ROOT_URL setting.
+    """
+    errors = []
+    if not getattr(settings, 'LMS_ROOT_URL', None):
+        errors.append(
+            Error(
+                'LMS_ROOT_URL is not defined.',
+                id='common.djangoapps.common_initialization.E001',
+            )
+        )
+    return errors
```

Comment:
Shouldn't this check go into the new `lms_initialization` app instead?

Reply:
Before my commit, this check was explicitly run in both the LMS and CMS startup.  I haven't looked into whether or not that's correct.